### PR TITLE
added prod task def, changed existing task to pre-prod and updated CI-CD

### PR DESCRIPTION
**Pull request by johnsimmons1980**

### CODE REVIEW

File: .github/workflows/ci-cd-workflow.yml [hunk 1]
```diff
@@ -190,3 +226,40 @@ jobs:
...
+          with:
+            task-definition: .aws/task-definition.json
+            container-name: ${{ github.event.repository.name }}
+            image: ${{ steps.login-ecr.outputs.registry }}/${{ github.event.repository.name }}:${{ github.sha }}
+
+        - name: Deploy Amazon ECS task definition
+          uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+          with:
+            task-definition: ${{ steps.task-def.outputs.task-definition }}
+            service: trainee-sync
```
Comment (JosephKelly) [inline .github/workflows/ci-cd-workflow.yml:263]: Copy Pasta, I guess.  Should be the `${{ github.event.repository.name }}` (starting with `tis-`).

File: .github/workflows/ci-cd-workflow.yml [hunk 2]
```diff
@@ -190,3 +226,40 @@ jobs:
...
+
+        - name: Log in to Amazon ECR
+          id: login-ecr
+          uses: aws-actions/amazon-ecr-login@v1
+
+        - name: Fill in the new image ID in the Amazon ECS task definition
+          id: task-def
+          uses: aws-actions/amazon-ecs-render-task-definition@v1
+          with:
+            task-definition: .aws/task-definition.json
```
Comment (JosephKelly) [inline .github/workflows/ci-cd-workflow.yml:255]: missing the `-prod`. Line 218 needs `-pre-prod` too.

**Review by JosephKelly — CHANGES_REQUESTED** [review]
A couple of things look like they need updating.

File: .github/workflows/ci-cd-workflow.yml (hunk 1 above)
Comment (johnsimmons1980) [reply .github/workflows/ci-cd-workflow.yml]: done

File: .github/workflows/ci-cd-workflow.yml (hunk 2 above)
Comment (johnsimmons1980) [reply .github/workflows/ci-cd-workflow.yml]: done

**Comment by johnsimmons1980** [comment]
think thats it on the updates

**Review by JosephKelly — APPROVED** [review]
LGTM :smile:

**Pull request merged**